src/components/PodcastCard/PodcastCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Image, Text, View} from 'react-native';
+import TitleCard from '../TitleCard/TitleCard';
+import styles from './PodcastCardStyle';
+import {podcast1, podcast2, podcast3} from '../../../assets';
+
+const PodcastCard = () => {
```

Comment:
Since you have two components that are very similar to each other, is it better to combine them?